### DISCUSSION TITLE
jupyter notebook에서 kiwipiepy import 실패

### DISCUSSION THREAD

**Discussion by circledal** [bab2min/kiwipiepy#148] [Q&A] [answered]
안녕하세요, kiwipiepy를 설치했는데도 불구하고 notebook에서 import가 안 되네요. 어떻게 해결하는지 알고 계신가요?
MacOS를 사용하고 있는데요 terminal에서는 "python -m kiwipiepy" 를 통해서 kiwipiepy 연습을 해도 문제없이 잘 작동되거든요. 
다시 kiwipiepy를 삭제했다가 설치해봐도 여전히 Jupyter Notebook에서는 kiwipiepy import가 안 되네요. ㅠㅠ 스크린샷 첨부합니다.. 파란색은 제 이름이라서 가렸어요.
<img width="1137" alt="스크린샷 2024-01-16 오후 2 13 37" src="https://github.com/bab2min/kiwipiepy/assets/156741753/58705004-8a76-4141-a16b-fd48aa8dd620">
<img width="1181" alt="스크린샷 2024-01-16 오후 2 13 18" src="https://github.com/bab2min/kiwipiepy/assets/156741753/c390ead1-cf22-4b79-9591-ce0bf9dc900b">

**Comment by bab2min** [comment, marked as answer]
안녕하세요~ @circledal 님, Kiwi에 관심을 가져주셔서 감사합니다.

올려주신 스크린샷을 참고하면, 현재 시스템에 크게 2종류의 Python(`~/opt/anaconda3/bin/python` 과 `~/miniconda3`)이 설치되어 있는걸로 보입니다. 그리고 현재 주피터 노트북을 구동하는 곳은 전자구요. 그런데 `pip install kiwipiepy`로 설치를 한 로그를 보면 miniconda3쪽에 패키지가 설치가 되었네요. 아마도 모종의 이유로 Python이 두 벌 설치되면서 pip가 꼬여서 pip 명령어가 miniconda3와 연결된 상황으로 보이네요. `In 7`에서 `sys.path.append`를 수행하여 miniconda3에 설치된 kiwipiepy를 가져오려고 하신듯한데 현재 구동중인 anaconda3의 Python 버전은 3.9이고 miniconda3의 Python 버전은 3.10이기 때문에 버전이 달라서 패키지를 가져올 수는 없습니다.
그래서 현재 사용중인 anaconda3 쪽에다가 kiwipiepy를 설치하거나, 주피터 노트북을 구동하는 Python을 miniconda3로 바꾸는 것으로 해결해야 할것 같습니다. 전자가 조금 더 간단할듯하니 그 방법을 공유해드릴게요.
```
!/Users/~~/opt/anaconda3/bin/pip install kiwipiepy
```
혹은
```
!/Users/~~/opt/anaconda3/bin/python -m pip install kiwipiepy
```
처럼 Python 경로를 직접 지정하여 kiwipiepy를 설치할 Python 환경을 명시해주는 걸 시도해보시길 바랍니다.

**Reply by circledal** [reply]
세상에.. 파이썬이 두 개가 설치가 되어서 그런 것이었군요! 친절한 답변 정말 감사합니다!
